source/parser.hera

```diff
@@ -1166,7 +1166,7 @@ BracedBlock
     }
 
 SingleLineStatements
-  ( TrailingComment* Statement ):first ( ( TrailingComment* Semicolon TrailingComment* ) Statement )*:rest ->
+  ( TrailingComment* Statement ):first ( ( TrailingComment* Semicolon TrailingComment* ) Statement? )*:rest ->
```

Comment:
`( TrailingComment* Semicolon TrailingComment* )` could be updated to `StatementDelimiter`

Reply:
`StatementDelimiter` includes `EOS` as a special case, which we probably don't want for `SingleLineStatements`. I could add an `ExplicitStatementDelimiter` perhaps?

Reply:
Good point, `ExplicitStatementDelimiter` or `SemicolonDelimiter` would work for the name of that common part.